### PR TITLE
Add Flint wallet support

### LINKED ISSUE #556: Add Flint wallet support

### PR DESCRIPTION
**Pull request by amirmrad**
Closes #556 .

### Pre-review checklist

- [x] All code has been formatted using our config (`make format`)
- [x] The integration and unit tests have been run (`npm run test`) locally
- [x] The changelog has been updated under the `## Unreleased` header, using the appropriate sub-headings (`### Added`, `### Removed`, `### Fixed`), and the links to the appropriate issues/PRs have been included

### CODE REVIEW

File: src/Wallet.purs [hunk 1]
```diff
@@ -67,10 +70,21 @@ mkGeroWalletAff = Gero <$> mkCip30WalletAff "Gero" _enableGero
...
+mkFlintWalletAff = Flint <$> mkCip30WalletAff "Flint" _enableFlint
+
+foreign import _enableFlint :: Effect (Promise Cip30Connection)
+
 cip30Wallet :: Wallet -> Maybe Cip30Wallet
 cip30Wallet = case _ of
   Nami c30 -> Just c30
   Gero c30 -> Just c30
+  Flint c30 -> Just c30
   _ -> Nothing
```
Comment (klntsky) [inline src/Wallet.purs:88]: Come to think of it, we should not use wildcard pattern here to make it harder to forget to add a new wallet when we do it.

**Comment by klntsky** [comment]
I am getting 

```
Uncaught Error: `submit` call failed. Error from Ogmios: [{"valueNotConserved":{"consumed":{"assets":{},"coins":0},"produced":{"assets":{"fda1b6b487bee2e7f64ecf24d24b1224342484c0195ee1b7b943db50.546865546f6b656e":100},"coins":1002989232}}},{"badInputs":[{"index":0,"txId":"962c7e6204e5b4e0e53b87f5ca10d0819af04d4cf97407d18982956a8da0a800"},{"index":1,"txId":"bc8dca6432b09b8c75b78462ae339707b6fa850c8d005f300df78c792484a427"},{"index":0,"txId":"c227876b250f21327a8f69ab5e8737136e5cbf188e89cda30404781195dcb947"}]}]

```

on http://localhost:8000/?SignMultiple:flint

**Comment by klntsky** [comment]
Probably a concurrent spending problem. I think I was running another example in parallel.

**Review by klntsky — APPROVED** [review]

**Review by ngua — CHANGES_REQUESTED** [review]
Please update the wallet list in the readme, otherwise LGTM

**Review by ngua — APPROVED** [review]

**Pull request merged**